ush/mesoscale/evs_sref_cnv.sh

```diff
@@ -125,9 +148,9 @@ echo "Print stat generation metplus log files end"
 #***********************************************
 # Gather small stat files to forma big stat file
 # **********************************************
-if [ $gather = yes ] ; then 
-  $USHevs/mesoscale/evs_sref_gather.sh $VERIF_CASE
-  export err=$?; err_chk
-fi
+#if [ $gather = yes ] ; then 
+#  $USHevs/mesoscale/evs_sref_gather.sh $VERIF_CASE
+#  export err=$?; err_chk
+#fi
```

Comment:
@BinbinZhou-NOAA Please consider removing this section of commented out code if it is no longer needed.

CC @PerryShafran-NOAA since this has to do with gathering small stat files.